src/dmtcp_launch.cpp

```diff
@@ -292,6 +296,9 @@ static void processArgs(int *orig_argc, char ***orig_argv,
     } else if (s == "--checkpoint-open-files" || s == "--ckpt-open-files") {
       checkpointOpenFiles = true;
       shift;
+    } else if (s == "--allow-file-overwrite") {
+      setenv(ENV_VAR_ALLOW_OVERWRITE_WITH_CKPTED_FILES, "1", 0);
```

Comment:
We should go over the list of cmdline flags and corresponding env vars to ensure they have "similar" strings.